jupyter-docker/Dockerfile

```diff
@@ -199,4 +199,4 @@ USER $USER
 WORKDIR $HOME
 
 EXPOSE $JUPYTER_PORT
-ENTRYPOINT ["jupyter notebook"]
+ENTRYPOINT ["/usr/local/bin/jupyter", "notebook"]
```

Comment:
We're overriding this anyway, but I found this was broken when I removed the override for testing. `ENTRYPOINT` needs to be in the format:
```
ENTRYPOINT ["executable", "param1", "param2"]
```